azavea/docker-django
# Bump Django versions to 2.0.13 + 1.11.20, remove 1.9 + 1.10

## Overview

* Update Django 2.0.x and 1.11.x
* Remove EOL 1.9 and 1.10 from the build matrix
* Update psycopg2 to latest patch release
* Fetch Alpine Postgres packages from `v3.8/main`

Closes #72 

## Notes

* There was a packaging error in planned release versions 2.0.11 and 1.11.19 ([ticket here](https://code.djangoproject.com/ticket/30175)), so this PR updates to the latest patch version that fixes those errors. Release notes:
    * https://docs.djangoproject.com/en/2.0/releases/2.0.11/
    * https://docs.djangoproject.com/en/2.0/releases/2.0.12
    * https://docs.djangoproject.com/en/2.0/releases/2.0.13/
    * https://docs.djangoproject.com/en/1.11/releases/1.11.19/
* Django 1.9 and 1.10 are EOL, so this PR removes them. See [the Django roadmap](https://www.djangoproject.com/weblog/2015/jun/25/roadmap/).
* See the [latest psycopg2 patch release notes](http://initd.org/psycopg/docs/news.html#what-s-new-in-psycopg-2-7-7) for info on what changed.
* As of Alpine Linux 3.9.0, Alpine no longer publishes Postgres 10.x packages. See [the 3.9 release notes](https://alpinelinux.org/posts/Alpine-3.9.0-released.html).

## Testing instructions

See the Travis CI build: https://travis-ci.org/azavea/docker-django/builds/491741160
